MaxText/ckpt_conversion/shape_mapping.py

```diff
@@ -0,0 +1,77 @@
+"""
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ """
+
+
+def GEMMA2_HF_WEIGHTS_TO_SHAPE_MAPPING(config):
```

Comment:
Thanks for the change! Do you think it's better we organize those files to something like bellow? I think we will onboard more ckpt conversion soon:

1) `ckpt_conversion` to host all model ckpt conversions;
2) put all utils like `utils.py`, `hf_utils.py` in `ckpt_conversion/utils`  folder or directly under `ckpt_conversion`;
3)  gemma2 specific files, like `convert_gemma2_to_hf.sh, param_mapping.py` move to `ckpt_conversion/gemma2` folder.
4) put test related files under `tests`, like `test_hf_ckpt.py`